src/Tribe/Views/V2/iCalendar/Links/Google_Calendar.php

```diff
@@ -115,7 +115,8 @@ public function generate_single_url( $post = null ) {
 			return '';
 		}
 
-		$event_details = empty( $event->description ) ? urlencode( $event->description ) : '';
+		$event_details = ! empty( $event->description ) ? $event->description : $event->post_content;
```

Comment:
Question: do we care that the event might have an empty `post_content`?

Reply:
Good point, we should cover that possibility thoroughly. I'll revise.